### PR TITLE
Fix/btc server/dup outputs

### PR DESCRIPTION
**Pull request by arminsabouri**

### CODE REVIEW

File: bin/btc-server/src/util.rs [hunk 1]
```diff
@@ -220,14 +231,11 @@ pub fn validate_psbt(
...
-            _ => return Err(ValidatePSBTError::FeeCalculationError(e)),
+            _ => return Err(ValidatePSBTError::PsbtError(e)),
         },
     };
 
-    // validate outputs cover all pegout ids or are change
-    if let Err(e) = validate_outputs(psbt, db) {
-        return Err(ValidatePSBTError::InvalidOutputs(e.to_string()));
-    };
+    println!("fee: {:?}", fee);
```
Comment (arminsabouri) [inline bin/btc-server/src/util.rs]: Whoops. I'll remove this

File: bin/btc-server/src/util.rs [hunk 2]
```diff
@@ -655,20 +664,11 @@ mod util_tests {
...
-        app.db.set_key_package(key_package.clone()).expect("set key package");
-
-        let _pegout_id = store_pending_pegout(&app.db);
-
-        app.db.store_utxos(&[&utxo]).unwrap();
-        app.db.flush().unwrap();
-
         psbt.inputs[0].set_eth_address(eth);
         let res = validate_psbt(&psbt, ROUND1, 2, &app.db);
+        println!("res: {:?}", res);
```
Comment (SatwikPrabhu) [inline bin/btc-server/src/util.rs]: Maybe this needs to be removed as well?

File: bin/btc-server/src/util.rs [hunk 3]
```diff
@@ -911,7 +916,62 @@ mod util_tests {
...
+            res.unwrap_err(),
+            ValidatePSBTError::InvalidOutputs(ValidateOutputsError::DuplicateOutputs)
+        );
+
+        // Its fine for the same exact output to the present multiple times, as a user can have
+        // multiple pegouts but they should have different pegout ids
+        let pegout_id2 = store_pending_pegout(&app.db);
+        dup_psbt.outputs[1].set_pegout_id(pegout_id2.as_bytes());
+        let res = validate_psbt(&dup_psbt, NO_FLAGS, app.min_signers, &app.db);
+        println!("res: {:?}", res);
```
Comment (SatwikPrabhu) [inline bin/btc-server/src/util.rs]: This as well?

**Review by scottmillner — APPROVED** [review]
thanks for catching this and checking for dups!

**Pull request merged**